### PR TITLE
[Doc] Translate tutorial 6_customize_runtime

### PR DESCRIPTION
**Pull request by Ben-Louis**
## Motivation

Improve the completeness of Chinese Documentation.

## Modification

Translate `docs/en/tutorials/6_customize_runtime.md` to its Chinese version in  `docs/zh_cn/tutorials/6_customize_runtime.md`.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmpose/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit tests to ensure correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

(I do not modify any code.)

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 1]
```diff
@@ -1,3 +1,342 @@
...
+
+如果要使用自己实现的新优化器 `MyOptimizer` ，可以进行如下修改：
+
+```python
+optimizer = dict(type='MyOptimizer', a=a_value, b=b_value, c=c_value)
+```
+
+### 自定义优化器构造器
+
+某些模型的优化过程可能对不同参数有不同的设置，例如批归一化层的权重衰减。
```
Comment (jin-s13) [inline docs/zh_cn/tutorials/6_customize_runtime.md:107]: 这两句感觉不是特别通顺

**Comment by ly015** [comment]
@Ben-Louis Please sign CLA.

File: docs/zh_cn/tutorials/6_customize_runtime.md (hunk 1 above)
Comment (ly015) [reply docs/zh_cn/tutorials/6_customize_runtime.md:107]: @Ben-Louis You can refer to https://github.com/open-mmlab/mmsegmentation/blob/master/docs/zh_cn/tutorials/customize_runtime.md

Comment (Ben-Louis) [reply docs/zh_cn/tutorials/6_customize_runtime.md]: 已参照 @ly015 贴的文档进行修改，目前表述为：”有些模型可能需要在优化器里对一些特别参数进行设置，例如批归一化层的权重衰减系数。“

**Review by jin-s13 — APPROVED** [review]

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 2]
```diff
@@ -1,3 +1,342 @@
...
+
+```python
+optimizer = dict(type='Adam', lr=0.001, betas=(0.9, 0.999), eps=1e-08, weight_decay=0, amsgrad=False)
+```
+
+### 使用自己实现的优化器
+
+#### 1. 定义一个新优化器
+
+如果您想添加一个新的优化器，名字叫`MyOptimizer`，参数包括 `a` 、 `b` 、 `c` ，可以按照以下步骤定义该优化器。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:52]:
```suggestion
如果您想添加一个新的优化器，名字叫`MyOptimizer`，参数包括 `a` 、`b` 、`c`，可以按照以下步骤定义该优化器。
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 3]
```diff
@@ -1,3 +1,342 @@
...
+
+首先，创建一个新目录 `mmpose/core/optimizer` 。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:54]:
```suggestion
首先，创建一个新目录 `mmpose/core/optimizer`。
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 4]
```diff
@@ -1,3 +1,342 @@
...
+
+    def __init__(self, a, b, c):
+
+```
+
+#### 2. 注册这个优化器
+
+新优化器必须先导入主命名空间才能被成功调用。有两种实现方式。
+
+- 修改`mmpose/core/optimizer/__init__.py`来导入
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:73]:
```suggestion
- 修改 `mmpose/core/optimizer/__init__.py` 来导入
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 5]
```diff
@@ -1,3 +1,342 @@
...
+      target_ratio=(0.85 / 0.95, 1),
+      cyclic_times=1,
+      step_ratio_up=0.4,
+  )
+  ```
+
+## 自定义训练策略
+
+我们默认使用的学习率变化策略为阶梯式衰减策略，即MMCV中的[`StepLRHook`](https://github.com/open-mmlab/mmcv/blob/f48241a65aebfe07db122e9db320c31b685dc674/mmcv/runner/hooks/lr_updater.py#L153)。
+此外，我们还支持很多[学习率变化策略](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/lr_updater.py)，例如余弦退火策略 `CosineAnnealing` 和多项式策略 `Poly` 。其调用方式如下
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:166]:
```suggestion
此外，我们还支持很多[学习率变化策略](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/lr_updater.py)，例如余弦退火策略 `CosineAnnealing` 和多项式策略 `Poly`。其调用方式如下
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 6]
```diff
@@ -1,3 +1,342 @@
...
+
+我们推荐用户在每轮训练结束后对模型进行评估，即采用 `EpochEvalHook` 工作流。不过很多用户仍采用 `val` 工作流。
+
+工作流是一个由（阶段，轮数）构成的列表，它规定了程序运行中不同阶段的顺序和轮数。默认的工作流为
+
+```python
+workflow = [('train', 1)]
+```
+
+即“训练1轮”。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:195]:
```suggestion
即“训练 1 轮”。
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 7]
```diff
@@ -1,3 +1,342 @@
...
+        pass
+
+    def before_iter(self, runner):
+        pass
+
+    def after_iter(self, runner):
+        pass
+```
+
+用户需要根据钩子的实际用途定义该钩子在 `before_run` 、 `after_run` 、 `before_epoch` 、 `after_epoch` 、 `before_iter` 以及 `after_iter` 中的行为。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:248]:
```suggestion
用户需要根据钩子的实际用途定义该钩子在 `before_run` 、`after_run` 、`before_epoch` 、`after_epoch` 、`before_iter` 以及 `after_iter` 中的行为。
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 8]
```diff
@@ -1,3 +1,342 @@
...
+
+用户可以通过将钩子的参数 `priority` 设置为 `'NORMAL'` 或 `'HIGHEST'` 来设定它的优先级
+
+```python
+custom_hooks = [
+    dict(type='MyHook', a=a_value, b=b_value, priority='NORMAL')
+]
+```
+
+钩子在注册时，其优先级默认为 `NORMAL` 。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:284]:
```suggestion
钩子在注册时，其优先级默认为 `NORMAL`。
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 9]
```diff
@@ -1,3 +1,342 @@
...
+
+- log_config
+- checkpoint_config
+- evaluation
+- lr_config
+- optimizer_config
+- momentum_config
+
+这些钩子中，只有日志钩子的优先级为 `VERY_LOW` ，其他钩子的优先级都是 `NORMAL` 。
+前面的教程已经讲述了如何修改 `optimizer_config` 、 `momentum_config` 、  `lr_config`。这里我们介绍如何修改 `log_config` 、 `checkpoint_config` 、 `evaluation`。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:308]:
```suggestion
这些钩子中，只有日志钩子的优先级为 `VERY_LOW`，其他钩子的优先级都是 `NORMAL`。
前面的教程已经讲述了如何修改 `optimizer_config` 、`momentum_config` 、`lr_config`。这里我们介绍如何修改 `log_config` 、`checkpoint_config` 、`evaluation`。
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 10]
```diff
@@ -1,3 +1,342 @@
...
+
+#### 模型权重文件配置
+
+MMCV的运行程序会使用 `checkpoint_config` 来初始化 [`CheckpointHook`](https://github.com/open-mmlab/mmcv/blob/9ecd6b0d5ff9d2172c49a182eaa669e9f27bb8e7/mmcv/runner/hooks/checkpoint.py#L9) 。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:312]:
```suggestion
MMCV的运行程序会使用 `checkpoint_config` 来初始化 [`CheckpointHook`](https://github.com/open-mmlab/mmcv/blob/9ecd6b0d5ff9d2172c49a182eaa669e9f27bb8e7/mmcv/runner/hooks/checkpoint.py#L9)。
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 11]
```diff
@@ -1,3 +1,342 @@
...
+```python
+checkpoint_config = dict(interval=1)
+```
+
+用户可以通过设置 `max_keep_ckpts` 来保存有限的模型权重文件；通过设置 `save_optimizer` 以决定是否保存优化器的状态。
+[这份文档](https://mmcv.readthedocs.io/en/latest/api.html#mmcv.runner.CheckpointHook)介绍了更多参数的细节。
+
+#### 日志配置
+
+日志配置 `log_config` 可以设置多个日志钩子，并且可以设定记录间隔。目前MMCV支持的日志钩子包括 `WandbLoggerHook` 、 `MlflowLoggerHook` 、 `TensorboardLoggerHook`。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:323]:
```suggestion
日志配置 `log_config` 可以设置多个日志钩子，并且可以设定记录间隔。目前MMCV支持的日志钩子包括 `WandbLoggerHook` 、`MlflowLoggerHook` 、`TensorboardLoggerHook`。
```

File: docs/zh_cn/tutorials/6_customize_runtime.md [hunk 12]
```diff
@@ -1,3 +1,342 @@
...
+    hooks=[
+        dict(type='TextLoggerHook'),
+        dict(type='TensorboardLoggerHook')
+    ])
+```
+
+#### 测试配置
+
+测试配置 `evaluation` 可以用来初始化 [`EvalHook`](https://github.com/open-mmlab/mmpose/blob/master/mmpose/core/evaluation/eval_hooks.py#L11)。
+除了参数 `interval` ，其他参数（例如 `metric`）会被传递给 `dataset.evaluate()`。
```
Comment (ly015) [inline docs/zh_cn/tutorials/6_customize_runtime.md:338]:
```suggestion
除了参数 `interval`，其他参数（例如 `metric`）会被传递给 `dataset.evaluate()`。
```

**Review by ly015 — APPROVED** [review]

**Pull request merged**